.github/workflows/build-mudlet.yml

```diff
@@ -42,7 +42,7 @@ jobs:
             buildname: 'windows'
             triplet: x64-mingw-dynamic
             # compiler: flag not used in windows pipeline
-            qt: '5.15.2'
+            qt: '5.14.1'
```

Comment:
So you have downgraded the GH Actions Windows Qt version from 5.15.2 to 5.14.1 but you still need to address the same issue with AppVeyor ...

Reply:
It's a different issue with appveyor.